types/circomlibjs/index.d.ts

```diff
@@ -0,0 +1,343 @@
+// Type definitions for circomlibjs 0.1
+// Project: https://github.com/iden3/circomlibjs
+// Definitions by: cedoor <https://github.com/cedoor>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Poseidon = (arr: any[], state?: any, nOut?: number) => any;
+
+export type Point = [any, any];
```

Comment:
I think you can constrain `export type Point = [any, any];`  with `export type Point = [Uint8Array, Uint8Array];`